lib/ship.rb

```diff
@@ -0,0 +1,21 @@
+require 'pry'
```

Comment:
I usually take out pry once I'm done testing with it, but not really a big deal if we keep it there